core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -51,37 +51,37 @@ object EntityComponent {
   val allAttributeValuesColumnSize = 65534
 }
 
+import slick.jdbc.MySQLProfile.api._
+
+sealed abstract class EntityTableBase[RECORD_TYPE <: EntityRecordBase](tag: Tag) extends Table[RECORD_TYPE](tag, "ENTITY") {
+  def id = column[Long]("id", O.PrimaryKey, O.AutoInc)
+  def name = column[String]("name", O.Length(254))
+  def entityType = column[String]("entity_type", O.Length(254))
+  def workspaceId = column[UUID]("workspace_id")
+  def version = column[Long]("record_version")
+  def deleted = column[Boolean]("deleted")
+  def deletedDate = column[Option[Timestamp]]("deleted_date")
+
+//  def workspace = foreignKey("FK_ENTITY_WORKSPACE", workspaceId, workspaceQuery)(_.id)
+  def uniqueTypeName = index("idx_entity_type_name", (workspaceId, entityType, name), unique = true)
+}
+
+class EntityTable(tag: Tag) extends EntityTableBase[EntityRecord](tag) {
+  def * = (id, name, entityType, workspaceId, version, deleted, deletedDate) <> (EntityRecord.tupled, EntityRecord.unapply)
+}
+
+class EntityTableWithInlineAttributes(tag: Tag) extends EntityTableBase[EntityRecordWithInlineAttributes](tag) {
+  def allAttributeValues = column[Option[String]]("all_attribute_values")
+
+  def * = (id, name, entityType, workspaceId, version, allAttributeValues, deleted, deletedDate) <> (EntityRecordWithInlineAttributes.tupled, EntityRecordWithInlineAttributes.unapply)
+}
+
```

Comment:
This chunk of code was pulled out of the EntityComponent trait below because I needed to reference it outside the trait. Unfortunately this means that I had to import MySQLProfile specifically because the usual way of `import driver.api._` requires `driver` to be in scope which ain't no more.